sim/pokemon.js

```diff
@@ -54,6 +54,7 @@ class Pokemon {
 		}
 		this.name = set.name.substr(0, 20);
 		this.speciesid = toId(this.species);
+		/**@type {Template} */ // Typescript bug fixed in 2.9
```

Comment:
I would rather depend on a beta version of TypeScript than leave a workaround in-code

Reply:
Yeah, that sounds like a better solution.